cumulusci/cli/cci.py

```diff
@@ -809,6 +807,19 @@ def org_connect(config, org_name, sandbox, login_url, default, global_org):
     oauth_dict = oauth_capture()
     org_config = OrgConfig(oauth_dict, org_name)
     org_config.load_userinfo()
+    org_config._load_orginfo()
+    if (
+        org_config.organization_sobject
+        and "TrialExpirationDate" in org_config.organization_sobject.keys()
+    ):
+        if org_config.organization_sobject["TrialExpirationDate"] is None:
+            org_config.config["expires"] = None
+        else:
+            org_config.config["expires"] = parse_api_datetime(
+                org_config.organization_sobject["TrialExpirationDate"]
+            )
+    else:
+        org_config.config["expires"] = "Persistent"
```

Comment:
I connected a persistent org but it shows as "Unknown" -- I think it ended up on line 816 because TrialExpirationDate was present but None. I think you have lines 816 and 822 reversed from where they should be.

Reply:
good catch